initial readme skeleton

kwohl opened the pull request:
git fetch --all
git checkout kw-readme

View preview of inital readme skeleton
Make sure steps to install and run the app are correct
Offer suggestions for things to add

kwohl committed:
initial readme skeleton

mKroogz commented:
This readme skeleton looks amazing!  Good job!

kwohl committed:
added icons to tech stack

kwohl committed:
added icons to read me

kwohl committed:
updated git ignore

alyssa-nycum-imprivata reviewed (changes requested):
This looks good so far! The only possible change I have is that in my other full-stack django REAMEs I have this as the command: "pip install -r requirements.txt"

I see you don't have the -r in there - but I'm not 100% sure if it's needed

kwohl committed:
fixed error in readme

kwohl committed:
updating readme

kwohl committed:
updated feature descriptions

kwohl committed:
description updates

kwohl committed:
VimCrypt~01!o½ª.iVå`g5!ê|r5ýö©tÅD	Û@ÈÏyǦUèQù39bÎå#G¾UUuÝ^û?î2E%p6©Z|9¬²ãP8°æQÛêØ¨ã
º
B`mD²b³GͽZÄ»éAáB¥àCTÓTØ(O´Ëg3¥ÁÍäÖbdÛ*ÀíY\hå>½y¾?eðÞl釟Ðöèáäûç±QTV²S¹\U$Zað­ÿL®±^äÎÏk²ïmäÛ%]ÑÃqkøgSÂ<Rrf/x3Yč
Ö{Ü*xsµn1m|¥¼)kü
@Á*إ_So=Éßý©Ë1)Xt<Ûb

kwohl committed:
fixed merge conflicts

alyssa-nycum-imprivata reviewed (approved):
You always make the best READMEs!!
